<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchProcessingDlg</class>
 <widget class="QDialog" name="BatchProcessingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пакетная обработка ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="sourceGrpBx">
       <property name="title">
        <string>Единый каталог исходных снимков</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QPushButton" name="sourceTlBtn">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icegui.qrc">
            <normaloff>:/dvg/icegui/images/open.png</normaloff>:/dvg/icegui/images/open.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="sourceLnEdt"/>
          </item>
          <item>
           <widget class="QLabel" name="extensionsLbl">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="distGrpBx">
       <property name="title">
        <string>Единый каталог обработанных снимков</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="distTlBtn">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icegui.qrc">
            <normaloff>:/dvg/icegui/images/open.png</normaloff>:/dvg/icegui/images/open.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="distLnEdt"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="addFilterTlBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icegui.qrc">
         <normaloff>:/dvg/icegui/images/addFilter.png</normaloff>:/dvg/icegui/images/addFilter.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delFilterTlBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icegui.qrc">
         <normaloff>:/dvg/icegui/images/delFilter.png</normaloff>:/dvg/icegui/images/delFilter.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="defaultPBtn">
       <property name="text">
        <string>Сброс</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useAllChBx">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="prBr">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="beginPBtn">
       <property name="text">
        <string>Обработать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mapPBtn">
       <property name="text">
        <string>Карта</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icegui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchProcessingDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchProcessingDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
